gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -64,6 +77,20 @@ export default function NxFormLayoutExample() {
         setFilePristine(false);
       };
 
+  const { initialState, userInput } = nxTextInputStateHelpers,
```

Comment:
Pull this up to where the imports are

Reply:
Updated here 8d3cb2ac97adaad4eccc60e1608d4414f357cb8b.